packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -315,7 +315,19 @@ class Auth {
   void _loadCredentials() {
     final envToken = platform.environment[shorebirdTokenEnvVar];
     if (envToken != null) {
-      _token = CiToken.fromBase64(envToken);
+      try {
+        _token = CiToken.fromBase64(envToken.trim());
+      } on FormatException catch (e) {
+        logger
+          ..err(
+            '''
+Failed to parse CI token from environment. This likely means that your CI token is incorrectly formatted.
```

Comment:
Should this say shorebirdTokenEnvVar or similar in the output?

Reply:
It could but that seems unnecessary–we only print this if we detect the environment variable

Reply:
We also do mention the variable name on the next line

Reply:
sg.  Just the person reading that message might not know what the env variable is 🤷